tests/ui_tests/app/api.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import allure
+import requests
+
+
+class RequestFailedException(Exception):
+    """Request to ADCM API has status code >= 400"""
+
+
+class APIRequester:
```

Comment:
Consider renaming this class to something more elegant. Like ADCMApiClient

Reply:
Good point, renamed to "ADCMDirectAPIClient" to distinct it from "ADCMClient" and keep ADCM and API terms cases readable